code/src/layouts/call-to-action/CallToAction.scss

```diff
@@ -0,0 +1,30 @@
+@import "@design-system";
+
+.call-to-action {
+  display: flex;
+  gap: spacing(10);
+  margin: spacing(10) 0;
+  flex-direction: column;
+
+  @include breakpoint("lg") {
+    flex-direction: row;
+  }
+
+  &__item {
+    flex-basis: 100%;
+    padding: spacing(14);
+    background-color: red;
```

Comment:
`red` use variables

Reply:
Thanks a lot @dmtrth25, i really forgot to remove it after some local testing. This was removed